packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.base.tsx

```diff
@@ -153,6 +153,15 @@ export class BreadcrumbBase extends BaseComponent<IBreadcrumbProps, any> {
   };
 
   private _onRenderItem = (item: IBreadcrumbItem) => {
+    const Tag = item.isCurrentItem && item.currentItemHeading ? item.currentItemHeading : 'span';
+    const breadcrumbItem = (): JSX.Element => (
+      <Tag className={this._classNames.item}>
+        <TooltipHost content={item.text} overflowMode={TooltipOverflowMode.Parent} {...this.props.tooltipHostProps}>
+          {item.text}
+        </TooltipHost>
+      </Tag>
+    );
+
     if (item.onClick || item.href) {
       return (
         <Link
```

Comment:
Seems like this should take effect regardless of `href` existing or not on `item`. I would expect an item to always be rendered as an `h*` if I specified it.

Reply:
@jdhuntington Agreed, I've adjusted the `<Link/>` component to also accept the heading prop so the user can have choose either headings with or without links but the heading will always be applied, thanks !
 
`<Link
          as={item.as ? item.as : 'a'}`